config.toml

```diff
@@ -3,11 +3,11 @@
 [components.celestia]
 
 [components.celestia.light]
-binary = "/tmp/vimana/celestia/celestia"
+binary = "/usr/local/bin/celestia/celestia"
 download = "/tmp/vimana/celestia/init.sh"
 
 [components.celestia.bridge]
-binary = "/tmp/vimana/celestia/celestia"
+binary = "/usr/local/bin/celestia/celestia"
 download = "/tmp/vimana/celestia/init.sh"
```

Comment:
I don't think this will work because the `/usr/local/bin/celestia` directory isn't created when sudo mv command executes in init.sh. We need to either create the directory or remove it from here.

Reply:
Is it ok to use the original path to put celestia?